critiquebrainz/frontend/views/place.py

```diff
@@ -35,6 +37,11 @@ def entity(id):
         limit=limit,
         offset=offset,
     )
+    try:
+        avg_rating = db_avg_rating.get(place['id'], "place")
+        avg_rating["rating"] = round(avg_rating["rating"] / 20, 1)
+    except db_exceptions.NoDataFoundException:
+        avg_rating = None
```

Comment:
This part of code is also almost the same among different views. Can it be moved into one function that does average rating retrieval?